src/js/view/view.js

```diff
@@ -181,6 +195,14 @@ function View(_api, _model) {
         }
     }
 
+    this.stopFloating = function () {
+        if (hasClass(_playerElement, 'jw-flag-floating')) {
```

Comment:
Once we implement max. 1 floating player per page, we can improve this check to not be dependent on the class.